modules/apps/headless/headless-user-notification/headless-user-notification-impl/rest-openapi.yaml

```diff
@@ -21,6 +21,11 @@ components:
                     format: date-time
                     readOnly: true
                     type: string
+                entryTitle:
```

Comment:
If this is the title of the entry, maybe rename it to title is more accurate?

Reply:
Q!
This field can be modified?
should be readonly?

Reply:
The choice of name was based on the existing field of the same name in the table, but I believe that a better name it is "comment" or "mention". What do you think?

Yes, in this case it would be `readonly`.